eventing/README.md

```diff
@@ -149,6 +149,8 @@ is created or updated.
   [ObjectReference](https://kubernetes.io/docs/reference/generated/kubernetes-api/v1.12/#objectreference-v1-core)
   A reference to the object that should receive events.
 
+Refer to [here](samples/kubernetes-event-source) for Kubernetes event source example.
```

Comment:
Global: Instead of "Refer to here," please rephrase to "See <name of sample> for an example."

Reply:
Thanks @samodell , Done.